oktapam/resource_security_policy.go

```diff
@@ -83,9 +87,48 @@ func resourceSecurityPolicy() *schema.Resource {
 							Description: descriptions.SecurityPolicyResources,
 							Elem: &schema.Resource{
 								Schema: map[string]*schema.Schema{
+									attributes.Secrets: {
+										Type:        schema.TypeList,
+										Optional:    true,
+										MinItems:    1,
+										MaxItems:    1,
+										Description: descriptions.SecurityPolicySecrets,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												attributes.Secret: {
+													Type:        schema.TypeList,
+													Optional:    true,
+													Description: descriptions.SecurityPolicySecret,
+													Elem: &schema.Resource{
+														Schema: map[string]*schema.Schema{
+															attributes.SecretID: {
+																Type:        schema.TypeString,
+																Required:    true,
+																Description: descriptions.SecretID,
+															},
+														},
+													},
+												},
```

Comment:
Can policy have multiple secret folders ?

Reply:
A policy, yes, but not in a single rule

Reply:
I've constrained this.